ray-operator/config/samples/ray-cluster.overwrite-command.yaml

```diff
@@ -0,0 +1,60 @@
+# This example config is used to describe how does the annotation "ray.io/overwrite-container-cmd" work.
+# See kuberay/docs/guidance/pod-command.md for more details.
```

Comment:
```suggestion
# See kuberay/docs/guidance/pod-command.md for more details.
```
Is this link still accurate? More generally, what's the plan to document this feature? (I guess we'll add a section in the doc in the Ray repo?)

Reply:
Good catch!

Reply:
> More generally, what's the plan to document this feature? (I guess we'll add a section in the doc in the Ray repo?)

Yes, I will update the doc later.

Reply:
Updated https://github.com/ray-project/kuberay/pull/1704/commits/5ad6b08647d260019bfa53b1904cc196dc63195a.